Source/WebCore/platform/graphics/cocoa/FontCacheCoreText.cpp

```diff
@@ -990,4 +1008,10 @@ void FontCache::platformReleaseNoncriticalMemory()
 #endif
 }
 
+std::optional<HashMap<String, URL>>& userInstalledFontMap()
```

Comment:
Returning a std::optional here doesn't seem terribly useful, A default constructed HashMap is just a null pointer, it's cheap to construct and it seems you can check if the HashMap is empty when needed?